_data/api/pro-sante-connect.md

```diff
@@ -0,0 +1,70 @@
+---
+title: API Pro Santé Connect
+tagline: Permettre l'authentification des professionnels de la santé
+external_site: https://esante.gouv.fr/
+is_open: -1 # -1 means API not open
+datapass_link: https://esante.gouv.fr/
```

Comment:
We can leave it empty for now, or comment. To comment, add a # at the beginning of the line